cassandra-3.11/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM openjdk:8u252-slim
 
-ARG VERSION=v0.0.5
+ARG STARGATE_VERSION=v0.0.5
```

Comment:
is this version hard-coded here? Or can it be overridden/parameterized?
If cannot be overridden it will be problematic because we would need to change this version quite often.

Reply:
Hardcoded and will be changed on each release. So when stargate releases v0.0.6 then you'll need a PR to this repo to deploy a new docker image for v0.0.6. This will provide audibility and gates for creating new docker releases.